pkg/cmd/roachprod/main.go

```diff
@@ -1089,6 +1089,15 @@ func main() {
 		fmt.Sprintf("The cloud provider(s) to use when creating new vm instances: %s", vm.AllProviderNames()))
 	createCmd.Flags().BoolVar(&createVMOpts.GeoDistributed,
 		"geo", false, "Create geo-distributed cluster")
+
+	defaultMap := make(map[string]string)
```

Comment:
I don't think this needs to be a separate variable. Simply pass in `make(map[string]string)` to the `StringToStringVar` invocation.